### PR TITLE
Increase test tolerances

### PR DESCRIPTION
**Pull request by gabrieltseng**
Increase test tolerances to make them less brittle

### CODE REVIEW

File: tests/integration/train/train_module/test_contrastive_latentmim.py [hunk 1]
```diff
@@ -208,8 +208,8 @@ def test_train_batch_with_missing_modalities(
         logger.info(mock_trainer._metrics)
         assert torch.allclose(
             mock_trainer._metrics["train/PatchDisc"],
-            torch.tensor(2.2),
```
Comment (favyen2) [inline tests/integration/train/train_module/test_contrastive_latentmim.py:211]: Can you add some comment about where the 2.1 comes from? I looked at the samples_with_missing_modalities fixture but it's not clear to me how to compute 2.1 from those values. Or maybe Henry can review this PR instead since he may be more familiar with the code here.

Comment (Hgherzog) [reply tests/integration/train/train_module/test_contrastive_latentmim.py:211]: It was just the loss computed from the random inputs, which is dependent on all the  randomness being properly seeded. The purpose was to catch anything that creates non sensical outputs like nan or something really high for our different train modules, and make sure that all the apis connections are happy for the different train modules. It seems like this broke on something we don't care about. I would suggest just checking like < 4 and is not nan or inf.

Comment (gabrieltseng) [reply tests/integration/train/train_module/test_contrastive_latentmim.py:211]: Sounds good. Should I update the other tests while I'm at it?

Comment (gabrieltseng) [reply tests/integration/train/train_module/test_contrastive_latentmim.py:211]: Updated

File: tests/integration/train/train_module/test_contrastive_latentmim.py [hunk 2]
```diff
@@ -174,16 +174,16 @@ def test_train_batch_without_missing_modalities(
...
-            torch.tensor(2.14),
-            atol=1e-1,
-        )
-        assert torch.allclose(
-            mock_trainer._metrics["train/InfoNCE"],
-            torch.tensor(0.196),
-            atol=1e-1,
-        )
+        loss = mock_trainer._metrics["train/PatchDisc"]
+        assert not torch.isinf(loss).any()
```
Comment (Hgherzog) [inline tests/integration/train/train_module/test_contrastive_latentmim.py]: nit: use a helper that checks these conditions in a single line assert

Comment (Hgherzog) [reply tests/integration/train/train_module/test_contrastive_latentmim.py]: re-use across tests

Comment (gabrieltseng) [reply tests/integration/train/train_module/test_contrastive_latentmim.py]: Added
